### PR TITLE
Fixturize and group fix

### PR DESCRIPTION
**Pull request by AlexandreJunod**

### CODE REVIEW

File: geocity/apps/accounts/admin.py [hunk 1]
```diff
@@ -663,6 +693,85 @@ def create_anonymous_user(self, request, administrative_entity_id):
...
+        elif request:
+            messages.add_message(request, messages.INFO, _("Rapport déjà existant."))
+
+        return redirect(
+            reverse(
+                "admin:forms_administrativeentityforadminsite_change",
+                kwargs={"object_id": administrative_entity_id},
+            )
+        )
+
```
Comment (monodo) [inline geocity/apps/accounts/admin.py:774]: I would suggest to move that part to the `Report `model

**Review by monodo — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -17,8 +17,16 @@ cp -n .env.example .env
 docker-compose up --build -d --remove-orphans
 # run the migrations
 docker-compose run web scripts/migrate.sh
-# load demo data
-docker-compose run web python manage.py fixturize_demo
+```
```
Comment (AlexandreJunod) [inline README.md:20]: remove

File: geocity/apps/forms/migrations/0015_merge_20230110_0919.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Generated by Django 4.1.4 on 2023-01-10 08:19
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("forms", "0008_alter_field_integrator_alter_form_integrator_and_more"),
```
Comment (AlexandreJunod) [inline geocity/apps/forms/migrations/0015_merge_20230110_0919.py:9]: prevent this migrations

File: geocity/apps/accounts/templates/accounts/admin/administrative_entity_change.html [hunk 4]
```diff
@@ -12,6 +12,14 @@
     <div class="form-group">
       <button formaction="{% url "admin:create_administrative_entity_anonymous_user" administrative_entity_id=original.pk %}" type="submit" class="btn btn-dark form-control">
         {% translate "Créer l'utilisateur anonyme" %}
+<<<<<<< HEAD
```
Comment (rbovard) [inline geocity/apps/accounts/templates/accounts/admin/administrative_entity_change.html]: Warning, merge issue...

**Pull request merged**